HowToRPC.md

```diff
@@ -0,0 +1,166 @@
+# How to Call Google APIs: RPC Edition
+
+_For many developers, the easiest way to call a Google API is with one of our
+client libraries. But occasionally someone may prefer to make API calls
+directly - perhaps from a language or environment that we don’t support or using
+a different networking library or tool. Here we’ll show you how to do it._
+
+This page focuses on calling Google APIs directly using their underlying RPC
+([Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call))
```

Comment:
[RPC](...)

People rarely use the term remote procedure call these days, so it is not useful to spell it out.

Reply:
@lukesneeringer had asked for this in an earlier comment:

| A paragraph saying "What is RPC?" would be valuable here.

I think spelling it out and linking to wikipedia is a very small (minimal?) way to provide that, so I'd prefer to keep this.

Reply:
Google search shows 35.5M use of RPC, only 824k use of "remote procedure call". Most people in the world don't speak native English. There is no point to force them to read more English words, like spelling out HTTP and RPC.